### DISCUSSION TITLE
Block Access to Tactical RMM via Public IP but Only Allow by DNS Name

### DISCUSSION THREAD

**Discussion by gtao725** [amidaware/tacticalrmm#1606] [General]
Hello Community,

Just wondering if it is possible to configure Tactical RMM so that access to the login page is only possible via DNS name and not via the public IP address? If so will such a change cause any issues with the way the entire system works? If nginx is modified to block access by direct ip, will an update reset the modification and needs to be applied again after each update? Thx very much in advance!

**Comment by wh1te909** [comment]
yes it's possible, TRMM only works via DNS anyway, if you try to use IP you'll get CORS errors. We usually don't touch nginx configs during updates except when necessary like to add a new route or something but it should not overwrite any changes you make.

**Reply by gtao725** [reply]
Thank you for the confirmation sir!
